Quality_declaration_fastcdr.md

```diff
@@ -0,0 +1,122 @@
+# **External Dependency Quality declaration** `fast-cdr` 
+
+This document is a declaration of software quality for the FastCDR external dependency, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
+
+As stated in their public repository, *eProsima FastCDR is a C++ library that provides two serialization mechanisms. One is the standard CDR serialization mechanism, while the other is a faster implementation that modifies the standard*.
+
+## Summary
+The `fast-cdr` library is used as one of the dependencies of the `fastrtps`, it is important for the eProsima team to consider this software to be reliable. On its current state, meets most of the requirements to qualify as a level 1 package.
+
+As for the missing topics, eProsima does not state their versioning policies for the repository and does not state explicitly the API/ABI policies for the code in  `fast-cdr`. As the code deals with the standard RPTS implementation, and it is used by their software `fastrtps` it is not expected that the basic API functionalities will be changed. In any case, the code is actively being tested against their  [CI](http://jenkins.eprosima.com:8080/) (both directly with the included test cases and indirectly through the tests of `fastrtps`)  and indirectly in the [ROS2 CI](https://ci.ros2.org/).
+
+There is no coverage information summary/report and, apparently, no checks being made in terms of linters/static analysis, however, as the core parts used in ROS2 are actively being tested under the packages that use  `fast-cdr`.
+
+Considering the previously mentioned reasons, especially the fact that this software is being tested both directly and indirectly with [CI jobs](http://jenkins.eprosima.com:8080/), we consider this library to be robust and reliable, and hence we declare it to qualify as a level 1 external dependency.
+
+## Comparison with ROS packages quality standards
+
+### Version policy
+
+ 1. *Must have a version policy*
```

Comment:
Use the Quality declaration template https://github.com/ros-infrastructure/rep/blob/4a3bbf6c296a725fda742ff8e695e2e1f6f59aa9/rep-2004.rst#quality-declaration-template